### ISSUE TITLE
Cross-node latency of about 7 seconds

### ISSUE THREAD

**Issue by OzySky** [chubaofs/chubaofs#1157]
Hi!
I am using Chubaofs as a RWX for k8s (deployed with Helm) and it's working great!
One thing I noticed is that when streaming (e.g a text file using node.js) to the volume, accessing the same file from a different node returns stale data for about 5-10 seconds after writing.
Any ideas?

**Comment by shuoranliu** [comment]
@OzySky There are several factors might affect the cross-node visibility latency:

1. **fsync of the writing node**: from performance perspective, the stream writing node updates the file meta upon fsync system call, or periodically. So in order for the other node to see the changes, the writing user can invoke fsync explicitly.
2. **AttrValid of reading node**: there is AttrValid time for the reading node, it won't send InodeGet request to metanode too frequently. Thus the reading node might see stale file size. However, newly opened files are guranteed to see latest file size and file meta information.

The bottom line is, the following process is guranteed for cross-node visibility: a. the writing user call fsync explicitly; b. then the reading user opens the file. This way, reading user is guarenteed to see latest contents.

Or you can decrease AttrValid time(there's a configuration in client options) to reduce the latency, in the cost of putting more pressure on metanode.
